doc/api/cluster.md

```diff
@@ -109,7 +109,8 @@ will be dropped and new connections will be refused. Node.js does not
 automatically manage the number of workers, however. It is the application's
 responsibility to manage the worker pool based on its own needs.
 
-
+Although the primary use for the cluster module is networking, this module
+can also be used for all other master/worker use-cases (without networking).
```

Comment:
Nit: remove "all" as certainly there are use cases where it is not the right choice.

I'd probably strip it down further to:

> Although a primary use case for the `cluster` module is networking, it can also be used for other use cases requiring worker processes.

Reply:
Thanks, sounds better indeed!